tests/BenchmarkDotNet.Tests/StringExtensionsTests.cs

```diff
@@ -20,5 +20,14 @@ public void AsValidFileNameDoesNotChangeValidFileNames()
 
             Assert.Equal(validFileName, validFileName.AsValidFileName());
         }
+
+        [Fact]
+        public void HtmlEncodeCharacters()
+        {
+            string expectedHtml = "&lt;&#39;&gt;&quot;&amp;shouldntchange";
+            string html = "<'>\"&shouldntchange";
+
+            Assert.Equal(html.HtmlEncode(), expectedHtml);

```

Comment:
nit: expected value should always be the first argument

thanks for adding the test!